### PR TITLE
Assign webhook URL to chart CRs even if they are not CP apps

### PR DESCRIPTION
**Pull request by tomahawk28**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: service/controller/app/resource/chart/desired.go [hunk 1]
```diff
@@ -74,17 +74,15 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 
 	annotations := generateAnnotations(cr.GetAnnotations())
 
-	if key.InCluster(cr) {
```
Comment (tomahawk28) [inline service/controller/app/resource/chart/desired.go:77]: We need to assign the webhook even if it's not CP apps.

File: helm/app-operator/templates/configmap.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ data:
         {{- if hasSuffix "-unique" .Release.Name }}
         webhookBaseURL: 'http://{{ include "resource.default.name" . }}.{{ include "resource.default.namespace" . }}.svc.cluster.local:{{ .Values.port }}'
         {{- else }}
-        webhookBaseURL: 'http://{{ include "resource.default.name" . }}.{{ .Values.Installation.V1.Kubernetes.API.Address }}'
+        webhookBaseURL: 'https://{{ include "resource.default.name" . }}.{{ .Values.Installation.V1.Kubernetes.API.Address }}'
```
Comment (tomahawk28) [inline helm/app-operator/templates/configmap.yaml:24]: It should be HTTPS, of course. I also test https schema with a service object, but it didn't work.

**Comment by tomahawk28** [comment]
Tested in `gauss`

**Review by rossf7 — APPROVED** [review]
LGTM

**Pull request merged**